examples/platform/silabs/wifi/wfx_rsi_host.cpp

```diff
@@ -15,6 +15,8 @@
  *    limitations under the License.
  */
 
+// MATTER INTERFACE
```

Comment:
maybe
```suggestion
// SL_MATTER WIFI INTERFACE
```

or something alike not just MATTER

Reply:
fixed